test/mainTest.js

```diff
@@ -22,7 +25,10 @@ describe('Main Test', () => {
                 process.exit();
             }, 10000);
         });
-        await fluxEndpointsTest(server);
-        await uiFluxEndpointsTest(server);
+        // await fluxEndpointsTest(server);
```

Comment:
should this be commented?

Reply:
Hi @TheTrunk on my recent changes with ui these are not commented. thanks